### ISSUE TITLE
Unexpected token in @urql/exchange-populate@0.2.1

### ISSUE THREAD

**Issue by protometa** [FormidableLabs/urql#1172]
urql version & exchanges: @urql/exchange-populate@0.2.1

**Steps to reproduce**

1. `npm i @urql/exchange-populate@0.2.1`
2. `node -r @urql/exchange-populate`

**Expected behavior**

Loads module

**Actual behavior**

```
SyntaxError: Unexpected token '}'
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at Module._preloadModules (internal/modules/cjs/loader.js:1217:12)
    at loadPreloadModules (internal/bootstrap/pre_execution.js:446:5)
    at prepareMainThreadExecution (internal/bootstrap/pre_execution.js:74:3)
    at internal/main/repl.js:19:1
```

**Comment by kitten** [comment]
I believe this is related to what's been fixed in https://github.com/FormidableLabs/urql/commit/17140762d786e68ea9b33d5019e30ace094045b4

This hot fix addressed a build issue that I only discovered after releasing several updates to packages which corrupted some releases. It seems I missed that it also affected `@urql/exchange-populate` so to fix a re-release is necessary.

That being said, no necessary changes to use the package were introduced and in the meantime `0.2.0` will still work.

So #1149 still takes precedence over this issue

**Comment by kitten** [comment]
This will be fixed by #1570 although usage of the `populateExchange` is not exactly recommended (and still experimental/early) until #964 lands.
